DEVELOPMENT.md

```diff
@@ -67,6 +67,138 @@ A `.clang-format` file which defines our specific code style lives in the
 top level directory of the repository. The style is identical to the Google
 style.
 
+## FAQ
+
+### What's the difference between `bootstrap-orbit.{sh,bat}` and `build.{sh,bat}`?
+
+`bootstrap-orbit.{sh,bat}` performs all the task which have to be done once per developer machine.
+This includes:
+* Installing system dependencies
+* Installing conan if necessary.
+
+Afterwards `bootstrap-orbit.{sh,bat}` calls `build.{sh,bat}`.
+
+`build.{sh,bat}` on the other hand performs all the tasks which have to be done
+once per build configuration. It creates a build directory, named after the
+given conan profile, installs the conan-managed dependencies into this build folder,
+calls cmake for build configuration and starts the build.
```

Comment:
This implies that when one changes the git branch, one may have to rerun build.sh, right? Maybe add a sentence about that?

Reply:
Yes. I added a sentence explaining that!